src/tests/specs/iteration.spec.ts

```diff
@@ -25,6 +25,15 @@ describe('Iteration test', () => {
     await planner.iteration.clickCreateIteration();
     expect(await planner.sidePanel.getIterationDate()).toContain('new Iteration [Active]'+month+' 1, '+year+' - '+month+' 28, '+year+'\n');
   });
+  
+  it('should create a new child iteration', async () => {
+   let newIteration = 'new child iteration';
+   let iteration3 = '/' + process.env.SPACE_NAME + '/Iteration_2';
+   await planner.sidePanel.createNewIteration();
+   await planner.iteration.addNewIteration(newIteration, iteration3);
+   await planner.sidePanel.clickExpander(iteration3);
```

Comment:
SidePanel will show only the iterationName not the space name..So usre clickExpander('Iteration_2')